dashboard/app/views/home/_nav.html.haml

```diff
@@ -5,10 +5,10 @@
 .row.navcontainer
   .span12{style: 'margin-left: 0'}
     %ul.nav.nav-pills{style: 'margin: 0'}
-      - if current_user && !current_user.working_on_user_scripts.empty?
+      - if current_user && !Queries::ScriptActivity.working_on_user_scripts(current_user).empty?
```

Comment:
Not strictly part of your refactor, but here and below I'd love to improve readability with
```suggestion
      - if current_user && Queries::ScriptActivity.working_on_user_scripts(current_user).any?
```
[Pretty sure that's equivalent.](https://apidock.com/ruby/Enumerable/any%3F#141-Using-any-on-Empty-Arrays-and-Hashes)